bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/io/ChannelPoolTest.java

```diff
@@ -17,25 +17,25 @@
 
 import static org.mockito.Matchers.any;
 import static org.mockito.Matchers.same;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
I removed the .*, but kept the import static.  For mockito, it's common practice.  For example, grpc does this quite a bit: https://github.com/grpc/grpc-java/search?utf8=%E2%9C%93&q=import+static+mockito